buildscripts/gateway-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+export SERVER_ENDPOINT=127.0.0.1:24240
+export ENABLE_HTTPS=0
+export ACCESS_KEY=minio
+export SECRET_KEY=minio123
+export MINIO_ACCESS_KEY=minio
+export MINIO_SECRET_KEY=minio123
+export AWS_ACCESS_KEY_ID=minio
+export AWS_SECRET_ACCESS_KEY=minio123
+
+./minio server data --address 127.0.0.1:24242 &
+sleep 3
+./minio gateway s3 http://127.0.0.1:24242 --address 127.0.0.1:24240 &
```

Comment:
We should actually log these console output in separate files, all logs will get clobbered in this format though.

Reply:
The right way to do this would be to redirect to a file, and print the files if error happens, is that what you were thinking too?

Reply:
yes @wlan0 - basically logs should be downloadable after a run, so it is better to put all of them together as artifacts and make them available to be downloaded.